mapbook-iOS.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,25 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "ArcGIS",
```

Comment:
My understanding is that because the Toolkit includes the SDK, we only need to include the Toolkit as a Swift Package and we'll get the SDK for "free".

Reply:
That's my understanding as well. Because it works either way, I suppose I was hoping to convey outwardly that there are two dependencies working in tandem. I also thought it might be nice to allow either dependency to be refined separately from one another, which would be beneficial if there are patch releases or if internal daily build support is introduced for SPM. Your thoughts?

Reply:
That's fine with me as long as there's no chance of the versions used being incompatible and there's no duplication of the SDK (once from the project SP and one from the Toolkit SP).

Reply:
> as long as there's no chance of the versions used being incompatible

The `Package.swift` guarantees this so long as the package file is configured accordingly. If they are incompatible, Xcode will display an error and the project won't build.

> there's no duplication of the SDK

Xcode downloads and manages dependencies in the Derived Data directory. In my testing it appears either project preference (passive vs active ArcGIS dependency preference) resolves the packages such that there is one ArcGIS binary stored in `artifacts` and one copy of ArcGISToolkit source code stored in `checkouts`. I think we're good here.